DESCRIPTION

```diff
@@ -51,7 +51,7 @@ Suggests:
     sf,
     snakecase,
     stringr,
-    tabulizer,
+    tabulapdf,
```

Comment:
Very annoyingly, `tabulizer` is fully deprecated and now replaced by `tabulapdf`. The changes in `sample_tax_bills_detail.R` are mostly related to accounting for the slight behavior difference between the two packages.